### PR TITLE
debian: add -unity package configuration

### LINKED ISSUE #1744: Can't build .deb package currently due to Unity assets
Trying to build a `.deb` of master (923fb20020846c56c74e0c72ff5044a672b5b169) fails with:

```
dh_missing --fail-missing
dh_missing: usr/share/themes/Yaru/unity/unmaximize_unfocused_pressed.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/unmaximize_unfocused_prelight.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/unmaximize_unfocused.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/unmaximize_focused_pressed.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/unmaximize_focused_prelight.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/unmaximize_focused_normal.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/unmaximize.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/minimize_unfocused_pressed.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/minimize_unfocused_prelight.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/minimize_unfocused.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/minimize_focused_pressed.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/minimize_focused_prelight.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/minimize_focused_normal.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/minimize.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/maximize_unfocused_pressed.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/maximize_unfocused_prelight.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/maximize_unfocused.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/maximize_focused_pressed.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/maximize_focused_prelight.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/maximize_focused_normal.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/maximize.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/launcher_icon_shine_54.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/launcher_icon_shine_150.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/launcher_icon_shadow_62.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/launcher_icon_shadow_200.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/launcher_icon_selected_back_54.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/launcher_icon_selected_back_150.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/launcher_icon_glow_62.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/launcher_icon_glow_200.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/launcher_icon_edge_54.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/launcher_icon_edge_150.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/launcher_icon_back_54.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/launcher_icon_back_150.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/close_unfocused_pressed.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/close_unfocused_prelight.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/close_unfocused.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/close_focused_pressed.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/close_focused_prelight.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/close_focused_normal.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru/unity/close.svg exists in debian/tmp but is not installed to anywhere
dh_missing: usr/share/themes/Yaru-dark/unity exists in debian/tmp but is not installed to anywhere
```

is there a plan wrt packaging these or should they be excluded from all debs for now?

### PR DESCRIPTION
**Pull request by clobrano**
closes #1744

### CODE REVIEW

**Comment by clobrano** [comment]
I am not sure about the dependency list, could you double check (or suggest me how to check myself)?

Thanks!

**Comment by clobrano** [comment]
It stops at GPG sign, hope it's enough
https://www.dropbox.com/s/0obm0ke1u04tio8/Interviewing%20Tips%20at%20Google.doc?dl=0

File: debian/yaru-theme-unity.install [hunk 1]
```diff
@@ -0,0 +1,2 @@
+usr/share/themes/Yaru/unity
```
Comment (didrocks) [inline debian/yaru-theme-unity.install]: For futur-proofing (snow theme? ;)), I would change this to `usr/share/themes/Yaru*/unity` and removing the following line. WDYT?

Comment (clobrano) [reply debian/yaru-theme-unity.install]: smart ;)

**Comment by iainlane** [comment]
On Wed, Jan 15, 2020 at 10:37:07AM -0800, Carlo Lobrano wrote:
> It stops at GPG sign, hope it's enough
> https://www.dropbox.com/s/0obm0ke1u04tio8/Interviewing%20Tips%20at%20Google.doc?dl=0

This is not the right document!

-- 
Iain Lane                                  [ iain@orangesquash.org.uk ]
Debian Developer                                   [ laney@debian.org ]
Ubuntu Developer                                   [ laney@ubuntu.com ]

**Comment by iainlane** [comment]
On Thu, Jan 16, 2020 at 01:12:08AM -0800, Carlo Lobrano wrote:
> > On Wed, Jan 15, 2020 at 10:37:07AM -0800, Carlo Lobrano wrote: It stops at GPG sign, hope it's enough https://www.dropbox.com/s/0obm0ke1u04tio8/Interviewing%20Tips%20at%20Google.doc?dl=0
> > This is not the right document!
> > […](#)
> > -- Iain Lane [ iain@orangesquash.org.uk ] Debian Developer [ laney@debian.org ] Ubuntu Developer [ laney@ubuntu.com ]
> 
> I already edited the message with the right document, but github didn't sent you a new email :D

Indeed, thanks. But anyway it looks like Didier is a better package
golfer than me, so I will leave him to do the merge.

-- 
Iain Lane                                  [ iain@orangesquash.org.uk ]
Debian Developer                                   [ laney@debian.org ]
Ubuntu Developer                                   [ laney@ubuntu.com ]

**Comment by clobrano** [comment]
Okay, could I delete the messages that expose a document of my Dropbox I didn't want to share? XD

**Comment by didrocks** [comment]
LGTM, thank you both!

**Pull request merged**